src/js/preloadWebComponents.js

```diff
@@ -7,7 +7,8 @@ function doesUserAgentContainString(str) {
 }
 
 // For information as to why this is needed, please see:
-// https://github.com/silvermine/videojs-chromecast/issues/17
+// https://github.com/silvermine/videojs-chromecast/issues/19
```

Comment:
Sorry just noticed this doing one last check... Shouldn't this be https://github.com/silvermine/videojs-chromecast/issues/17?

Reply:
Hmm... I don't remember changing that.... Yes it should be #17